src/braket/circuits/angled_gate.py

```diff
@@ -87,6 +88,16 @@ def bind_values(self, **kwargs):
         """
         raise NotImplementedError
 
+    def adjoint(self) -> List[Gate]:
```

Comment:
This is true for all of our current gates but might not be in the future. Is there a way to catch it?

Reply:
This is the default behavior, and future gates could just override it.